lms/static/sass/elements/_controls.scss

```diff
@@ -403,3 +403,22 @@
       margin-bottom: none;
     }
   }
+
+  .btn-link {
```

Comment:
Ah ok, I think I know why this is happening. I'm ok with this for now, and I can add in a follow-on story that reviews. Thanks!